Makefile

```diff
@@ -2,10 +2,11 @@ init:
 	@cp ./web-app/.env.dist ./web-app/.env
 	@cp ./.docker/pgadmin/.env.pgadmin.dist ./.docker/pgadmin/.env.pgadmin
 	@cp ./.docker/db/.env.db.dist ./.docker/db/.env.db
+	@cp ./mobile-app/.env.dev.dist ./mobile-app/.env
```

Comment:
To be consistent with other we could remove the .dev/.prod to keep only one file !